app/src/main/cpp/Controller.h

```diff
@@ -45,6 +45,9 @@ struct Controller {
   vrb::TogglePtr modelToggle;
   vrb::TogglePtr handMeshToggle;
   std::vector<vrb::TransformPtr> handJointTransforms;
+  ControllerDelegate::HandMesh handMesh;
+  bool hasHandMesh;
```

Comment:
Why don't we store the hand mesh in an `unique_ptr` and create it on demand?

It has also the advantage of not having to add the extra `hasHandMesh` because you could perform null checks

Reply:
Right, I thought about but since I don't haven't use unique_ptr before, I felt conservative :). Will change that.